auth/test/test_basic.py

```diff
@@ -13,6 +12,7 @@
 )
 from auth.validateresult import AuthKind, ValidateResult
 from data import model
+from test.fixtures import *
```

Comment:
Is this extra import used?

Reply:
Yes, otherwise the test will fail with errors about fixtures not being found. Other tests in this directories import the fixtures the same way.